robot/framesystem/service_test.go

```diff
@@ -145,19 +146,19 @@ func TestFrameSystemFromConfig(t *testing.T) {
 
 	// There is a point at (1500, 500, 1300) in the world referenceframe. See if it transforms correctly in each referenceframe.
 	worldPose := referenceframe.NewPoseInFrame(referenceframe.World, spatialmath.NewPoseFromPoint(r3.Vector{1500, 500, 1300}))
-	armPt := r3.Vector{0, 0, 500}
+	armPt := r3.Vector{500, 0, 0}
```

Comment:
Is there a place we can check where these values are coming from?

Reply:
Unfortunately not really, this was all hand calculated.  But you can see the kinematic information in `fake_model.json`.  Does this answer your question?